packages/shared/src/utils/constants/contract.tsx

```diff
@@ -319,71 +319,71 @@ export const getMarketNamesMap = () => ({
 
 export const getUnsupportedContracts = () => ({
     EXPIRYMISS: {
-        name: <Localize i18n_default_text='Ends Outside' />,
```

Comment:
The changes in this file are not needed, please revert.

Reply:
Thank you for noticing, but if I leave `<Localize />`, then I need to extract trade_type_neme this way: `getUnsupportedContracts()[unsupported_contract_type]?.name.props.i18n_default_text`
If I replace `<Localize />` by `localize`, then it looks much better:
`getUnsupportedContracts()[unsupported_contract_type]?.name`
This replacement wouldn't affect files, where getUnsupportedContracts is called